lib/Doctrine/Migrations/Version/Executor.php

```diff
@@ -345,10 +345,11 @@ private function executeVersionExecutionResult(
 
             $this->outputSqlQuery($key, $query);
 
+            // Do not use method that returns results here, to not block next queries
```

Comment:
You can use methods that returns results, that's exactly what you did. For that reason, `executeQuery` should not be changed to `executeUpdate`
```suggestion
            // executeQuery() must be used here because $query might return a result set, for instance REPAIR does
```